components/TransactionAction.jsx

```diff
@@ -0,0 +1,145 @@
+let React = require('react');
+
+let classNames = require('classnames');
+let accounting = require('accounting');
+
+let ReactBootstrap = require('react-bootstrap');
+let Grid = ReactBootstrap.Grid;
+let Row = ReactBootstrap.Row;
+let Col = ReactBootstrap.Col;
+let Input = ReactBootstrap.Input;
+let Button = ReactBootstrap.Button;
```

Comment:
From `react-bootstrap` actually, an independent project which may not be strong enough for production use. Reminds me `Angular UI` which is a nightmare because when there is a bug in it you're f**cked.